### ISSUE TITLE
Postgresql 12 not working fine with the last downloaded jdbc driver

### ISSUE THREAD

**Issue by EmmanuelVinet33** [TOSIT-IO/tdp-getting-started#205]
Postgresql 12 is specified in ansible script but the last jdbc driver is downloaded. This result in some errors accessing dbs, during installation.
Changing postgresl 12 version by 15 version works finer.

**Comment by rpignolet** [comment]
Last JDBC driver should work with all PostgreSQL version, what is the JDBC version which does not work ?

**Comment by EmmanuelVinet33** [comment]
42.5.0
The last one. The ansible script is downloading this one.
I verified in the postgres db and all tables and contents seems to be there but acessing through the driver a part of script don't retrieve relations.

**Comment by rpignolet** [comment]
How do you verified the postgres db and tables contents ? Tables are created inside schema, not in the public one, your `search_path` is correctly configured ?

**Comment by EmmanuelVinet33** [comment]
I entered in the vagrant VM and connect as postgres user on the db. 
All seems to be correct in the DB but during Hive installation particularly relations are not found.
I had 2 options, either try with an older driver or change the postgresql version, what I did.
With the last version of postgresql, TDP installation went to the end.
We've tried the TDP installation on 3 different machines with 16 cores and 32Gb or RAM on Ubuntu 22.04. On these 3 machines we have the same problems.
